SPECS/kernel-hyperv/config

```diff
@@ -717,10 +718,6 @@ CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
 # end of GCOV-based kernel profiling
 
 CONFIG_HAVE_GCC_PLUGINS=y
-CONFIG_GCC_PLUGINS=y
```

Comment:
The GCC plugins missing/disabled is odd to see. We use these plugins for hardening the kernel. Not necessarily a blocker for the merge but something we need to revisit

Reply:
I'm not sure yet why these are turned off with the new compiler. Definitely will follow up though to re-enabled after merging